api/shared/const.go

```diff
@@ -66,3 +66,4 @@ var FunctionTimeoutInMillis = int64(3000)
 
 const FileServiceMaxBufSize = 2048
 const FileServicePathPrefix = "/parigot/app/"
+const FileServiceMaxPathPart = 20
```

Comment:
20 seems like a good choice.  More than that is probably an attack on our service.